sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.dto.ts

```diff
@@ -119,6 +138,22 @@ export class CreateApplicationOfferingChangeRequestAPIInDTO {
   reason: string;
 }
 
+/**
+ * Details to update the application offering change request by student.
+ */
+export class StudentApplicationOfferingChangeRequestAPIInDTO {
+  @IsOptional()
```

Comment:
IsOptional must be removed as ValidateIf will take care of validation based on condition.